README.md

```diff
@@ -26,15 +26,15 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
-1. What is the DOM?
+1. What is the DOM? The DOM is the "document object model," or the model of the object you are working on. For example, when working in a browser, the DOM refers to all the code that exists within the browser. 
```

Comment:
The DOM is an object representation of your HTML. The DOM can be modified using Javascript.